sample/build.gradle.kts

```diff
@@ -12,6 +12,7 @@ android {
   defaultConfig {
     minSdk = 19
     targetSdk = 30
+    multiDexEnabled = true
```

Comment:
Why was multidex enabled? Is the sampel app too big after the dependency changes?

If that's the case, it's probably easier to just bump the minSdk version to `21` and forget about multidex.

Reply:
I did not know that using minSdk into 21 would fix that issue, if it does then I will bump the minSdk version and remove multidex!

Reply:
Yes please, that would be great. Min SDK 21 is good enough for the sample.